x/concentrated-liquidity/pool_test.go

```diff
@@ -1058,93 +1089,3 @@ func (s *KeeperTestSuite) TestPoolExists() {
 	// ensure that this returns false
 	s.Require().False(poolExists)
 }
-
-// func (s *KeeperTestSuite) TestCalcInAmtGivenOut() {
```

Comment:
I'm guessing that these are re-introduced in: https://github.com/osmosis-labs/osmosis/pull/3535?

Reply:
Yep!